src/Lib/Items/CanvasImage.vala

```diff
@@ -103,6 +105,30 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
         });
     }
 
+    private bool is_width_small (int width, int height) {
+        // Callculate the width if height was 1
```

Comment:
Typo for "Calculate".
Also, this sentence is not grammatically correct and it doesn't make sense.
What are you trying to say here?